python/ray/train/session.py

```diff
@@ -501,3 +529,40 @@ def train_func():
     if session is None:
         return 1
     return session.world_size
+
+
+def get_accelerator(default_accelerator_cls: Type[Accelerator]) -> Accelerator:
+    """The accelerator for this training session.
+
+    If an accelerator has not been set, then this method will construct an
+    accelerator using the provided accelerator class.
+
+    Raises:
+        RuntimeError: if the session is uninitialized.
+    """
+    session = get_session()
+    if session is None:
+        raise RuntimeError(
+            "The function you called is meant to be used inside a training function "
+            "that is executed by `Trainer.run`."
+        )
+    return session.get_accelerator(default_accelerator_cls)
```

Comment:
A minor point, but what do you think about doing this instead 

```
if not session.accelerator:
    accelerator = default_accelerator_cls()
    session.accelerator = accelerator
return session.accelerator
```

Then session doesn't need to implement its own getter and setter.